### PR TITLE
1146 results sorting

### PR DESCRIPTION
**Pull request by slawosz**
* Adds sorting to search results
* As a additional work, webpacker was fixed

Screenshot from 2021-07-01 13-22-44

### CODE REVIEW

**Review by nasirkhanbeis — COMMENTED** [review]
Sort by relevance, newest and oldest looking good. 

spotted few issues  around image upload
1. When a large size file image is uploaded eg. 14MB then in search domain thumbnail image is broken
2. Non-js browser returns  500 error when 30MB file is upload, this is intermittent and happens once in 2-3 times. (looks like timing out in backend). Expected to throw error in the UI when more than 30mb file is uploaded
3. No feedback when a file is being parsed/processed in backend, when I click continue button several times it is returning 404 page.

File: .github/workflows/rubocop.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
     - name: Set up Nodejs
       uses: actions/setup-node@v1
       with:
-        node-version: 10.24.x
+        node-version: 14.17.x
```
Comment (scruti) [inline .github/workflows/rubocop.yml:43]: :partying_face:

File: cosmetics-web/Gemfile [hunk 2]
```diff
@@ -41,7 +41,7 @@ gem "sidekiq-cron", "~> 1.1.0"
 gem "slim-rails", "~> 3.2.0"
 gem "slowpoke", "~> 0.3.0"
 gem "strong_migrations", "~> 0.5.1"
-gem "webpacker", "~> 4.2.2"
+gem "webpacker", "~> 5.0"
```
Comment (scruti) [inline cosmetics-web/Gemfile:44]: :man_dancing:

File: cosmetics-web/app/models/elasticsearch_query.rb [hunk 3]
```diff
@@ -1,9 +1,17 @@
 class ElasticsearchQuery
-  def initialize(keyword:, category:, from_date:, to_date:)
+  SCORE_SORTING = "score".freeze
+  DATE_ASCENDING_SORTING  = "date_ascending".freeze
+  DATE_DESCENDING_SORTING = "date_descending".freeze
+
+  # AVAILABLE_SORTING = [SCORE_SORTING, DATE_ASCENDING_SORTING, DATE_DESCENDING_SORTING]
```
Comment (scruti) [inline cosmetics-web/app/models/elasticsearch_query.rb:6]:
```suggestion
```

**Review by scruti — APPROVED** [review]

**Review by nasirkhanbeis — APPROVED** [review]
Tested latest changes, looks good as expected. Noticed some issues in relation to image upload which are raised in separate cards.

**Pull request merged**